deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -14,8 +14,24 @@ namespaceOverride: ""
 ##
 fullnameOverride: ""
 
+## Configuration for installing openebs-monitoring dashboards in already installed prometheus stack
+customPrometheusStack:
```

Comment:
This doesn't seem readable friendly. Can this be changed to something - `openebsMonitoringComponents`?